Purchases/Purchasing/RCStoreKitRequestFetcher.m

```diff
@@ -24,8 +20,8 @@ - (SKReceiptRefreshRequest *)receiptRefreshRequest
 
 @end
 
-@interface RCStoreKitRequestFetcher ()
-@property (nonatomic) RCProductsRequestFactory *requestFactory;
```

Comment:
renamed since it no longer produces ProductsRequests. Also its name wasn't accurate before, since it's also able to produce `SKRefreshReceiptRequests`